libs/ui-toolkit/src/components/tabs/tabs.tsx

```diff
@@ -18,36 +18,39 @@ export const Tabs = ({ children }: TabsProps) => {
       className="h-full grid grid-rows-[min-content_1fr]"
       onValueChange={(value) => setActiveTab(value)}
     >
-      <div className="bg-black-10 dark:bg-white-10">
+      <div className="bg-black-10 dark:bg-black-70">
         <TabsPrimitive.List
-          className="flex flex-nowrap gap-4 overflow-x-auto"
+          className="flex flex-nowrap overflow-x-auto"
           role="tablist"
         >
-          {Children.map(children, (child) => {
-            if (!isValidElement(child)) return null;
-            const isActive = child.props.id === activeTab;
-            const triggerClass = classNames(
-              'py-4 px-20',
-              'capitalize',
-              'focus-visible:outline-none focus-visible:shadow-inset-vega-pink dark:focus-visible:shadow-inset-vega-yellow',
-              {
-                'font-semibold text-vega-pink dark:text-vega-yellow': isActive,
-                'bg-white dark:bg-black': isActive,
-                'text-black dark:text-white': !isActive,
-                'bg-white-90 dark:bg-black-70 hover:bg-white-95 dark:hover:bg-black-80':
-                  !isActive,
-              }
-            );
-            return (
-              <TabsPrimitive.Trigger
-                data-testid={child.props.name}
-                value={child.props.id}
-                className={triggerClass}
-              >
-                {child.props.name}
-              </TabsPrimitive.Trigger>
-            );
-          })}
+          <div className="gap-4 bg-white dark:bg-black flex border-r-4 border-r-white dark:border-r-black">
+            {Children.map(children, (child) => {
+              if (!isValidElement(child)) return null;
+              const isActive = child.props.id === activeTab;
+              const triggerClass = classNames(
+                'py-4 px-20',
+                'capitalize',
+                'focus-visible:outline-none focus-visible:shadow-inset-vega-pink dark:focus-visible:shadow-inset-vega-yellow',
+                {
+                  'text-vega-pink dark:text-vega-yellow [-webkit-text-stroke-width:1px]':
```

Comment:
Chore (blocking): Hmm, webkit-text-stroke-width is non-standard and shouldn't be used in production. We should find another way to highlight the tab instead of using weights/stroke that'll work in all modern browsers. Suggestions: underline, bg colour

Reply:
Thanks Sam. I've removed text-stroke and replaced it with another solution - I've added a pseudo-element to the inactive tab state which contains the same text in bold but is set to `visibility: hidden`. Then there's no change in width when it switches to the active state. This achieves the same result with standard CSS - hopefully it's OK!